app/src/main/java/de/westnordost/streetcomplete/quests/air_pump/AddBicyclePump.kt

```diff
@@ -0,0 +1,36 @@
+package de.westnordost.streetcomplete.quests.air_pump
+
+import de.westnordost.streetcomplete.R
+import de.westnordost.streetcomplete.data.osm.mapdata.Element
+import de.westnordost.streetcomplete.data.osm.mapdata.MapDataWithGeometry
+import de.westnordost.streetcomplete.data.osm.mapdata.filter
+import de.westnordost.streetcomplete.data.osm.osmquests.OsmFilterQuestType
+import de.westnordost.streetcomplete.data.osm.osmquests.Tags
+import de.westnordost.streetcomplete.data.user.achievements.QuestTypeAchievement.BICYCLIST
+import de.westnordost.streetcomplete.quests.YesNoQuestAnswerFragment
+import de.westnordost.streetcomplete.util.ktx.toYesNo
+
+class AddBicyclePump : OsmFilterQuestType<Boolean>() {
+
+    override val elementFilter = """
+        nodes, ways with
+         amenity = bicycle_repair_station or shop = bicycle
+         and !compressed_air
+         and !service:bicycle:pump
+    """
+    override val changesetComment = "Add whether bicycle pump is available"
+    override val wikiLink = "Key:service:bicycle:pump"
+    override val icon = R.drawable.ic_quest_air_pump
+    override val questTypeAchievements = listOf(BICYCLIST)
+
+    override fun getTitle(tags: Map<String, String>) = R.string.quest_air_pump_bicycle_title
+
+    override fun getHighlightedElements(element: Element, getMapData: () -> MapDataWithGeometry) =
+        getMapData().filter("nodes, ways with compressed_air or amenity = compressed_air or service:bicycle:pump")
```

Comment:
I would highlight all `amenity=bicycle_repair_station` and all `amenity=fuel` in addition to that.

This would make easier to handle cases with multiple repair stands.

(are standalone `amenity = compressed_air` without name rendered properly?)

Reply:
> (are standalone `amenity = compressed_air` without name rendered properly?)

If by "properly" you mean as an magenta dot, then yes, for example https://www.openstreetmap.org/node/2496359080

Reply:
good point about extra highlighting, added